src/utils/cmakeUtil.mts

```diff
@@ -15,7 +15,9 @@ import { buildCMakeIncPath } from "./download.mjs";
 
 export const CMAKE_DO_NOT_EDIT_HEADER_PREFIX =
   // eslint-disable-next-line max-len
-  "== DO NEVER EDIT THE NEXT LINES for Raspberry Pi Pico VS Code Extension to work ==";
+  "== DO NOT EDIT THE FOLLOWING LINES for the Raspberry Pi Pico VS Code Extension to work ==";
+export const CMAKE_DO_NOT_EDIT_HEADER_PREFIX_OLD =
+  "== DO NEVER EDIT THE NEXT LINES for the Raspberry Pi Pico VS Code Extension to work ==";
```

Comment:
This line should be the same as before (ie without the "the"), and also needs max-len disabled to remove the linting warning

```suggestion
// eslint-disable-next-line max-len
  "== DO NEVER EDIT THE NEXT LINES for Raspberry Pi Pico VS Code Extension to work ==";
```